spec/components/lead_providers/table_spec.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+RSpec.describe LeadProviders::YourSchools::Table, type: :view_component do
+  let(:schools) { Kaminari.paginate_array(Array.new(rand(5..10)) { |i| double("School #{i}") }).page(1) }
```

Comment:
What is `double` doing here?
Also, I get a bit lost around second `{` here - can we do it with `do-end` syntax and some intermediate steps?

Reply:
`double` is a handy object meaning "I do not really care what it is in this test". All I care here is that whatever it is, it is being passed to a TableRow.

I think I have refactored it in the next branch, I'll bring it here. I also consider if I shouldn't just accept a scope of schools here and paginate it inside the component?

Reply:
We can pass the page as an attribute. Just wrote that and it looks pretty solid to me :)